Userland/Libraries/LibJS/Runtime/ExecutionContext.h

```diff
@@ -48,7 +48,7 @@ struct ExecutionContext {
     DeprecatedFlyString function_name;
     Value this_value;
     MarkedVector<Value> arguments;
-    MarkedVector<Value> local_variables;
+    size_t local_variable_count { 0 };
```

Comment:
it seems very confusing that ExecutionContext is aware of number of local variables although with these changes CallFrame owns them.

Reply:
I agree, but I am not sure where to put them otherwise, could maybe put them on the CallFrame if that sounds better and is feaseable

Reply:
Sounds like it is worth looking into whether we should try join CallFrame with ExecutionContext.